.travis.yml

```diff
@@ -82,13 +102,22 @@ matrix:
       - JOB_TAG=_NUMPY_DEV
       - NOSE_ARGS="not slow and not network and not disabled"
       - PANDAS_TESTING_MODE="deprecate"
+      addons:
+        apt:
+          packages:
```

Comment:
so some of the builds are showing up above the ``allow_failures`` section. We want to keep the first 5 builds (osx, 2 x 2.7, 3.4, 3.5) as the required builds, the rest as allow_failures (mainly so things don't take so long, not because they are actually allowed to fail).

So you need to repeat anything you added on in the declaration as below; e.g. for the NUMPY_DEV build, you added:

```
addons:
  apt:
    packages:
       ...
```

just repeat this in the below section (so they are identical), then it will be fixed.